build-system/tasks/check-sourcemaps.js

```diff
@@ -33,6 +33,7 @@ const sourcemapUrlMatcher =
 // Mapping related constants
 const expectedFirstLineFile = 'src/polyfills/array-includes.js';
 const expectedFirstLineCode = 'function includes(value, opt_fromIndex) {';
```

Comment:
General comment: While you're here, can we better document all these constants so that future maintainers of this file have an easier time? (I admit I didn't do a great job while writing this check.)

```js
expectedFirstLineFile // First file that's compiled into v0.js
expectedFirstLineCode // First line of code in that file
expectedFirstLineColumn // Column at which the first line is found (after an initial 'use strict;')
```